src/AppBundle/Model/EventWiki.php

```diff
@@ -75,14 +76,21 @@ class EventWiki
     protected $stats;
 
     /**
-     * A bzcompressed string of all the page IDs. Compression is used so we don't have to store enormous strings.
-     * When pulled from the database, this may be of type resource and not string. Use self::getPages() to get an
-     * array representation of the page IDs.
+     * A bzcompressed string of all the IDs of pages created. Compression is used so we don't have to store enormous
+     * strings. When pulled from the database, this may be of type resource and not string. Use self::getPages() to get
+     * an array representation of the page IDs.
      * @see https://secure.php.net/manual/en/function.bzcompress.php
-     * @ORM\Column(name="ew_pages", type="blob", length=512, nullable=true)
+     * @ORM\Column(name="ew_pages_created", type="blob", length=512, nullable=true)
```

Comment:
I don't remember why I added `length=512`. From the migration you generated it seems this has no effect, but we should still remove it for clarity. Doesn't have to be with this PR.

Reply:
Oh yes, that's a good point. I'll remove it now. Blobs can't have length; maybe it was to be used for an index length? But we're not indexing these so it doesn't matter.